robottelo/utils/__init__.py

```diff
@@ -0,0 +1,42 @@
+# Independent utility functions that doesnt need separate module and
+import base64
+
+from cryptography.hazmat.backends import default_backend as crypto_default_backend
+from cryptography.hazmat.primitives import serialization as crypto_serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+
+
+def gen_ssh_keypairs():
```

Comment:
Why not in `robottelo.ssh`?

Reply:
The `robottelo.ssh` is host based module like firing command on some host. But this function is a private and public SSH Key generator function which is non-host based. By adding this to the function ssh module, I dont want people to feel its generating ssh keypair for any specific host.